R/http-handlers.R

```diff
@@ -187,7 +187,7 @@ register_http_handlers <- function(session, metadata) {
     settings <- utils::rc.settings()
     utils::rc.settings(ops = TRUE, ns = TRUE, args = TRUE, func = FALSE,
                        ipck = TRUE, S3 = TRUE, data = TRUE, help = TRUE,
-                       argdb = TRUE, fuzzy = FALSE, files = TRUE, quotes = TRUE)
+                       argdb = TRUE, fuzzy = FALSE, files = TRUE, quotes = FALSE)
```

Comment:
Let's also set `files` to `FALSE`, even though it's deprecated.

```suggestion
                       argdb = TRUE, fuzzy = FALSE, files = FALSE, quotes = FALSE)
```